selfdrive/ui/qt/widgets/tsk_keyboard.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <QPushButton>
+#include <QFile>
+
+#include "system/hardware/hw.h"
+
+#ifdef SUNNYPILOT
+#include "selfdrive/ui/sunnypilot/qt/widgets/controls.h"
+#define ButtonControl ButtonControlSP
+#define ToggleControl ToggleControlSP
+#else
+#include "selfdrive/ui/qt/widgets/controls.h"
+#endif
```

Comment:
I may need to clean up the #includes a bit. I'm new to cpp so wasn't sure what to remove